### ISSUE TITLE
application with no hostname produces an invalid manifest

### ISSUE THREAD

**Issue by glyn** [cloudfoundry/cli#767]
If an application is pushed with no hostname:
```
cf push appnohost -b https://github.com/cloudfoundry/staticfile-buildpack.git --no-hostname -d example.com
```
and then its manifest is captured:
```
cf create-app-manifest appnohost
```
The resultant manifest is invalid:
```
$ cf push -f ./appnohost_manifest.yml 
FAILED
Error reading manifest file:
host should not be null
```

The manifest looks like this:
```
$ cat appnohost_manifest.yml 
---
applications:
- name: appnohost
  memory: 1024M
  instances: 1
  buildpack: https://github.com/cloudfoundry/staticfile-buildpack.git
  host: 
  domain: example.com
```
It should use the [no-hostname attribute](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#nohosts).

**Comment by dkoper** [comment]
Thanks @glyn & @benpatt!
I have reproduced it, we'll investigate.

**Comment by glyn** [comment]
Great. Thanks.

**Comment by dkoper** [comment]
This has been implemented in cf CLI v6.16.0.
https://github.com/cloudfoundry/cli#downloads

Regards,
Dies Koper
CF CLI PM

**Comment by glyn** [comment]
Excellent.
